plugins/reporters/evaluated-model/src/funTest/assets/evaluated-model-reporter-test-deduplicate-expected-output.yml

```diff
@@ -65,59 +65,80 @@ issue_resolutions:
   comment: "A comment explaining why the issue can be ignored."
 issues:
 - _id: 0
+  timestamp: "2024-04-22T10:36:10.661544294Z"
```

Comment:
> Add one issue to an included- and another to an excluded project

"Add one issue to an included and another one to an excluded project"

> This prepares adding the ability

"This prepares for adding the ability" or "This prepares to add the ability"